### PR TITLE
Submarine-463. [WEB]Implement database connection in project page with Angular

### PR DESCRIPTION
**Pull request by chiajoukuo**
### What is this PR for?
Implement database connection in project page with Angular

### What type of PR is it?
[Feature]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-463

### How should this be tested?
https://travis-ci.org/github/apache/submarine/builds/688270358

### Screenshots (if appropriate)

workspaceIT

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by chiajoukuo** [comment]
@liuxunorg  @kevin85421  @jasoonn  @kobe860219 Can you help me review the pull request? Thanks!

**Comment by jasoonn** [comment]
@chiajoukuo Thank for your contribution. It looks good to me.

**Comment by kobe860219** [comment]
@chiajoukuo Thanks for your contribution. This pr is looks good to me!

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/project/new-project-page/new-project-page.component.ts [hunk 1]
```diff
@@ -84,10 +107,25 @@ export class NewProjectPageComponent implements OnInit {
...
+      permission: this.newProjectContent.permission,
+      starNum: 0,
+      likeNum: 0,
+      messageNum: 0
+    }
+    console.log(project)
+    // this.projectService.addProject(project).subscribe(() => {
+    // }, err => {
+    //   console.log("ERROR", err)
+    // });
```
[the remark this answers is not in the document]
Comment (chiajoukuo) [reply submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/project/new-project-page/new-project-page.component.ts:127]: Ok. I'll remove them!

**Comment by chiajoukuo** [comment]
@jasoonn @pingsutw @liuxunorg Thanks for your advice. I have updated my pr.

**Comment by jasoonn** [comment]
@chiajoukuo Thank for your contribution. This pr looks good to me.

**Comment by liuxunorg** [comment]
Will merge if no more comments.

**Pull request closed without merging**